.github/workflows/build_and_test_workflow.yml

```diff
@@ -6,9 +6,9 @@ name: Build and test
 # trigger on every commit push and PR for all branches except feature branches and pushes for backport branches
 on:
   push:
-    branches: [ '**', '!feature/**', '!backport/**' ]
+    branches: ['**', '!backport/**']
   pull_request:
-    branches: [ '**', '!feature/**' ]
+    branches: ['**']
```

Comment:
Can we accommodate other feature branches that do not expect to have CI enabled?

Reply:
Other feature branch will not be affected, because the change is not submitted to `main`. It will only work on `feature/multi-datasource` branch.

Reply:
Yes, let's ensure that this file is not included when we merge to main.